apps/files/src/components/FileLinkForm.vue

```diff
@@ -1,104 +1,107 @@
-            <input type="text" class="uk-input" :class="{ 'uk-form-danger': !$_expirationIsValid }" v-model="expireDate" :placeholder="placeholder.expireDate" />
-        </div>
-        <div class="uk-width-1-1 uk-width-3-5@m">
-            <label class="uk-form-label" for=""><span v-translate>Password</span><em v-if="$_passwordEnforced" class="uk-margin-small-left">(<span v-translate>required</span>)</em></label>
-            <input type="password" class="uk-input" :class="{ 'uk-form-danger': !$_passwordIsValid }" v-model="password" :placeholder="hasPassword && password === null? '********' : placeholder.password"/>
-        </div>
-        </div>
-        <!-- @TODO: Enable Mail API to use the following
-                    ++++++++++++++++++++++++++++++++++++
-        <template v-if="$_sendMailEnabled">
-            <h4 class="uk-margin-medium-top uk-heading-divider">
-                Send mail notification
-            </h4>
-            <div class="uk-margin">
-                <input type="text" class="uk-input" :placeholder="placeholder.mailTo" />
-            </div>
-            <div class="uk-margin">
-                <textarea class="uk-textarea" :placeholder="placeholder.mailBody rows="4"></textarea>
-            </div>
-            <div class="uk-margin">
-                <label><input type="checkbox" class="uk-checkbox uk-margin-small-right" v-translate>Send a copy to myself</label>
-            </div>
-        </template>
-        -->
-        <div class="uk-text-right">
-          <oc-button :disabled="linksLoading" @click="$_closeForm">Cancel</oc-button>
-          <oc-button v-if="!linksLoading && $_isNew" :disabled="!$_isValid" variation="primary" @click="$_addLink" v-translate>Create</oc-button>
-          <oc-button v-else-if="!linksLoading && !$_isNew" :disabled="!$_isValid" variation="primary" @click="$_updateLink" v-translate>Save</oc-button>
-          <button v-else disabled class="uk-button uk-button-default uk-position-relative"><oc-spinner class="uk-position-small uk-position-center-left" size="small" />
-            <span class="uk-margin-small-left" v-if="$_isNew" v-translate>Creating</span>
-            <span class="uk-margin-small-left" v-else v-translate>Saving</span>
-          </button>
-        </div>
-    </div>
+  <div class="oc-files-file-link-form uk-background-muted uk-padding-small">
+    <transition name="custom-classes-transition" enter-active-class="uk-animation-slide-top-small" leave-active-class="uk-animation-slide-top-small uk-animation-reverse">
+      <div class="uk-alert-danger" v-if="errors" uk-alert>
+        <a class="uk-alert-close" uk-close></a>
+        <p v-text="errors" />
+      </div>
+    </transition>
+    <div class="uk-margin">
+      <label class="uk-form-label" v-translate>Name your link</label>
+      <input class="uk-input" v-model="name" />
+      </div>
+      <h4 class="uk-margin-medium-top uk-heading-divider">
+        <template v-translate>Set Roles</template>
```

Comment:
You can use directly `<translate>` as a tag. No need for the template. Or even better `<translate tag="h4">` but almost no one does it so that's not such an issue :grin: